ramls/traits/includePackages.raml

```diff
@@ -3,6 +3,5 @@ queryParameters:
     displayName: Nested resource
     type: string
     description: Name of nested resource to include
-    default: packages
```

Comment:
I removed this "default:", because default should be null in this case, if default is packages then we always get include=packages flag